lifecycle-application/src/test/java/io/quarkus/ts/openshift/lifecycle/LifecycleApplicationOpenShiftIT.java

```diff
@@ -0,0 +1,42 @@
+package io.quarkus.ts.openshift.lifecycle;
+
+import io.fabric8.kubernetes.api.model.PodList;
+import io.fabric8.openshift.client.OpenShiftClient;
+import io.quarkus.ts.openshift.common.OpenShiftTest;
+import io.quarkus.ts.openshift.common.injection.TestResource;
+import org.junit.jupiter.api.Test;
+
+import static io.restassured.RestAssured.when;
+import static org.hamcrest.CoreMatchers.containsString;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+@OpenShiftTest
+public class LifecycleApplicationOpenShiftIT {
+
+    private static final String DEFAULT_ARGUMENT = "-Djava.util.logging.manager=org.jboss.logmanager.LogManager";
```

Comment:
this looks suspicious, is `-Djava.util.logging.manager=org.jboss.logmanager.LogManager` added to java command before `-jar ....` or after?

Reply:
I'm not sure what's your point here. Here, we're verifying that `@CommandLineArguments` is injecting the argument properly and these arguments are provided by the OpenShift extension.

Reply:
What's the command executed on OpenShift?

Is it `java -Djava.util.logging.manager=org.jboss.logmanager.LogManager -jar quarkus.jar` or is it `java -jar quarkus.jar -Djava.util.logging.manager=org.jboss.logmanager.LogManager` ?

Reply:
Oh got it. Thanks for spotting it. 
I've changed this test case to actually assert that the logging property is not an argument.